person.rb

```diff
@@ -1,14 +1,15 @@
 require './nameable'
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
Please add both `Getters` and `Setters` for @rentals in the `Person` class.

Reply:
@SajjadAhmad14 agian rentals is an array there is no need to define it as a Setter